modules/homa-lite/src/lib.rs

```diff
@@ -267,6 +281,23 @@ pub mod module {
 			}
 			current_weight
 		}
+
+		fn on_initialize(n: T::BlockNumber) -> Weight {
+			// Update the total amount of Staking balance by acrueing the interest periodically.
+			let interest_rate = Self::staking_interest_rate_per_update();
+			if !interest_rate.is_zero()
+				&& n.checked_rem(&T::StakingUpdateFrequency::get())
+					.unwrap_or_else(One::one)
+					.is_zero()
+			{
+				let current = Self::total_staking_currency();
+				let new_total = current.saturating_add(interest_rate.mul(current));
+				let _ = Self::update_total_staking_currency_storage(new_total);
```

Comment:
add comment explain why it is ok to ignore the result
in this case, it can only fail if current is zero, which is ok to fail

Reply:
Added comment